build.gradle

```diff
@@ -58,20 +58,16 @@ java {
 
 apply plugin: 'maven-publish'
 
-task exportDependenciesToTeamCity
-
 artifactory {
-    def baseUrl = System.getenv('ARTIFACTORY_URL') ?: project.properties.get('artifactoryUrl')
-    if (baseUrl != null) {
-        contextUrl = "${baseUrl}/artifactory" as String
-    }
-
     publish {
+        def baseUrl = System.getenv('ARTIFACTORY_URL') ?: project.properties.get('artifactoryUrl')
+        if (baseUrl != null) {
+            contextUrl = "${baseUrl}/artifactory" as String
+        }
         repository {
-            repoKey = 'rnd-maven-dev-local'
+            repoKey = System.getenv('ARTIFACTORY_DEPLOYER_REPO_KEY') ?: project.findProperty('NEXUS_REPO_KEY')
```

Comment:
ARTIFACTORY_DEPLOYER_REPO_KEY -> ARTIFACTORY_REPOSITORY_KEY
NEXUS_REPO_KEY is not needed